controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -286,39 +241,11 @@ func (r *StorageConsumerReconciler) get(obj client.Object) error {
 	return r.Client.Get(r.ctx, key, obj)
 }
 
-func (r *StorageConsumerReconciler) delete(obj client.Object) error {
-	if err := r.Client.Delete(r.ctx, obj); err != nil && !errors.IsNotFound(err) {
-		return err
-	}
-	return nil
-}
-
-// Checks whether a string is contained within a slice
-func Contains(slice []string, s string) bool {
-	for _, item := range slice {
-		if item == s {
-			return true
-		}
-	}
-	return false
-}
```

Comment:
I believe this is used in multiple places (It is a public function) Are we sure it is ok to remove this?

Reply:
Yes, it was throwing golangci lint error also, I make ocs-operator and no error so we are good